PrimerSDK/Classes/User Interface/Primer/SingleCardIconComponent.swift

```diff
@@ -7,6 +7,7 @@
 
 import UIKit
 
+// FIXME: Unused?
 class SingleCardIconComponent: UIView {
```

Comment:
same for this (remove if lint says not used).

Reply:
Removed